docs/release-process.md

```diff
@@ -88,6 +92,7 @@ Run twine upload to the pypi test repository:
 ```
 twine upload --repository pypitest dist/*
 ```
+The twine install location may not be on PATH by default; either add it or call twine using its full path.
```

Comment:
you can also do "python twine upload <args>"... I think

Reply:
Oh ok, I'll try that next time!